src/main/java/Task.java

```diff
@@ -0,0 +1,20 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "\u2718" : " "); //if done return X
```

Comment:
If you're just returning an X, would it be better to just use 'X' so that the confusing unicode can be avoided?